src/Sylius/Behat/Page/Admin/Order/ShowPage.php

```diff
@@ -234,75 +234,91 @@ public function hasTax($tax)
     }
 
     /**
-     * @param string $itemName
-     *
-     * @return string
+     * {@inheritdoc}
      */
     public function getItemUnitPrice($itemName)
     {
         return $this->getItemProperty($itemName, 'unit-price');
     }
 
     /**
-     * @param string $itemName
-     *
-     * @return string
+     * {@inheritdoc}
      */
     public function getItemDiscountedUnitPrice($itemName)
     {
         return $this->getItemProperty($itemName, 'discounted-unit-price');
     }
 
     /**
-     * @param string $itemName
-     *
-     * @return string
+     * {@inheritdoc}
      */
     public function getItemQuantity($itemName)
     {
         return $this->getItemProperty($itemName, 'quantity');
     }
 
     /**
-     * @param string $itemName
-     *
-     * @return string
+     * {@inheritdoc}
      */
     public function getItemSubtotal($itemName)
     {
         return $this->getItemProperty($itemName, 'subtotal');
     }
 
     /**
-     * @param string $itemName
-     *
-     * @return string
+     * {@inheritdoc}
      */
     public function getItemDiscount($itemName)
     {
         return $this->getItemProperty($itemName, 'discount');
     }
 
     /**
-     * @param string $itemName
-     *
-     * @return string
+     * {@inheritdoc}
      */
     public function getItemTax($itemName)
     {
         return $this->getItemProperty($itemName, 'tax');
     }
 
     /**
-     * @param string $itemName
-     *
-     * @return string
+     * {@inheritdoc}
      */
     public function getItemTotal($itemName)
     {
         return $this->getItemProperty($itemName, 'total');
     }
-    
+
+    /**
+     * {@inheritdoc}
+     */
+    public function hasPaymentPrice($price)
```

Comment:
Does it make sense to have this method with an argument? I'll comment the reason at its usages.